docs/templates/language-reference.md

```diff
@@ -44,20 +44,45 @@ Ideally, also with a full working example that shows it in context.
 
 Depending on the language, a brief description of the recommended way to handle dependencies in the language.
 
-## Connect to services
+## Connecting to services
 
-If available, include examples for connecting in an app
+If available, include examples for connecting to different services in an app
 (taken from https://examples.docs.platform.sh/).
 
-### Configuration reader
+The following examples show how to use <LANGUAGE_NAME> to access various services. To configure a given service, see the page dedicated to that service.
 
-If available, include the configuration reader available for the given language
-(for example, for [Python](https://github.com/platformsh/config-reader-python)).
+{{< codetabs >}}
+
+---
+title=<SERVICE>
+file=static/files/fetch/examples/<LANGUAGE_TYPE>/<SERVICE>
+highlight=<LANGUAGE_TYPE>
+markdownify: false
+---
+
+<--->
+
+---
+title=<SERVICE>
+file=static/files/fetch/examples/<LANGUAGE_TYPE>/<SERVICE>
+highlight=<LANGUAGE_TYPE>
+markdownify: false
+---
+
+{{< /codetabs >}}
+
+<!-- If available, include the configuration reader available for the given language,
+ by adding the shortcode below and providing the link to the specific flavour of the config reader
+(for example, for the Python library, the config reader link is: [platformshconfig](https://github.com/platformsh/config-reader-python)). -->
```

Comment:
:lady_beetle: This is not what I meant. Our general pattern with variables is to show the command with the variables and then show an example with actual values _after_ that command. Adding all of the additional text here doesn't seem to add as much clarity.